README.md

```diff
@@ -69,25 +69,25 @@ Kokkos Remote Spaces is built using [CMake](https://cmake.org) version 3.17 or l
 Building with `SHMEM`
 ```bash
    $: cmake . -DKRS_ENABLE_SHMEMSPACE=ON
-           -DKokkos_DIR=${KOKKOS_BUILD_DIR}
-           -DSHMEM_ROOT=${PATH_TO_MPI}
-           -DCMAKE_CXX_COMPILER=mpicxx
+           -DKokkos_DIR=${Kokkos_ROOT}
```

Comment:
The README already says that `Kokkos_ROOT` is the "Path to the root of the Kokkos install", so I figured that would be consistent.  But I'm fine either way, would you like me to make the change to `/path/ro/kokkos/install/dir` @dalg24?

I think I understand the hesitation to suggest using env vars.  I'll remove that.

Reply:
Also @dalg24, are you suggesting users of the OpenSHMEM backend should always set `SHMEM_ROOT` to MPI and the compiler wrapper to `mpicxx`?  I don't think that will ever work except with OpenMPI... unless I'm missing something.

On a related note, I'm not sure why OpenMPI doesn't include `oshcc` and `oshc++` symlinks/wrppers - I believe it's required by the OpenSHMEM spec (v1.5 Annex B).

Reply:
The issue is that `-DKokkos_DIR=<...>` is being used instead of `-DKokkos_ROOT=<...>`.

Not suggesting anything about SHMEM, I am not familiar with it like I am with Kokkos.

Reply:
Ah, of course.  Good catch!

Reply:
And @dalg24 do you have a preference regarding `/path/to/kokkos/install/dir` versus `${KOKKOS_INSTALL_DIR}`?  Seems like the latter may be more consistent with the existing README, but doesn't matter to me.